io/plugins/eu.esdihumboldt.hale.io.shp.ui/META-INF/MANIFEST.MF

```diff
@@ -24,5 +24,9 @@ Import-Package: com.google.common.io;version="9.0.0",
  eu.esdihumboldt.util
 Require-Bundle: org.eclipse.core.runtime;bundle-version="3.7.0",
  org.eclipse.ui;bundle-version="3.7.0",
- eu.esdihumboldt.hale.io.shp;bundle-version="2.5.0"
+ eu.esdihumboldt.hale.io.shp;bundle-version="2.5.0",
+ org.geotools,
```

Comment:
Are all these additional imports necessary?

Reply:
No need. I just update the file.